### PR TITLE
2963 themes

### PR DESCRIPTION
**Pull request by easherma**
# Description

... Please include a summary of the change and what it fixes.

Fixes # https://github.com/cityofaustin/techstack/issues/2963

# Testing Notes

Colors should reflect on local, staging, or PR, but not on Prod (you could test this by inverting the flag or merging to prod)

# Checklist:
- [ ] Request reviewers
- [ ] Slack-out message for review
- [ ] Link this PR in the issue
- [ ] Moved card to *review* in zenhub

### CODE REVIEW

File: joplin/base/context_processors.py [hunk 1]
```diff
@@ -2,4 +2,7 @@
 
 
 def settings_context(request):
```
Comment (easherma) [inline joplin/base/context_processors.py]: The context processor is Django's preferred way of passing constants into the app for templates

File: joplin/templates/wagtailadmin/base.html [hunk 2]
```diff
@@ -10,7 +10,14 @@
 
 
 {{ block.super }}
+
+<script>
```
Comment (easherma) [inline joplin/templates/wagtailadmin/base.html]: passing the constant from context to a javascript variable as a boolean

File: joplin/templates/wagtailadmin/login.html [hunk 3]
```diff
@@ -1,3 +1,11 @@
 {% extends "wagtailadmin/login.html" %}
+{% load render_bundle from webpack_loader %}
 
-{% block branding_login %}Welcome to Joplin!{% endblock %}
+{% block branding_login %}
+{% if ISPRODUCTION == False %}
```
Comment (easherma) [inline joplin/templates/wagtailadmin/login.html]: render 'theme' bundle (which is literally just an import of a scss file atm, but could be expanded in the future)

File: joplin/webpack.base.js [hunk 4]
```diff
@@ -14,6 +14,9 @@ module.exports = {
     editor: path.resolve(__dirname, "./js/editor.js"),
     createContentModal: path.resolve(__dirname, "./js/CreateContentModal/index.js"),
     janisBranchSettings: path.resolve(__dirname, "./js/janisBranchSettings.js"),
+
+    theme: path.resolve(__dirname, "./js/theme.js"),
+
```
Comment (Niicck) [inline joplin/webpack.base.js]: No need for newlines here.

**Pull request merged**